File: WAR/public/WAR.cpp
#include "judge.h"

class WAR : public iWAR {
public:
  virtual void init(int workshopsNum, int maxParticipants) {}
  virtual int registration(int byteid, char level[], int preferences[]) { return 0; }
  virtual int unregistration(int byteid) { return 0; }
};

int main() {
  WAR w;
  judge::run(&w);
}

File: WAR/public/WAR.java
public class WAR {
	public void init(int workshopsNum, int maxParticipants) {}
	public int registration(int byteid, char[] level, int[] preferences) { return 0; }
	public int unregistration(int byteid) { return 0; }

	public static void main(String[] args) {
		Judge.run(new WAR());
	}
}

